ia/src/utils.py

```
@@ -0,0 +1,11 @@
+import sys
+
+def exit_error(message):
```

Ruff (0.11.9)

Missing return type annotation for public function `exit_error`

Add return type annotation: `None`

(ANN201)